.github/workflows/ci.yml

```diff
@@ -28,6 +28,9 @@ jobs:
           - name: awx-collection
             command: /start_tests.sh test_collection_all
             label: Run Collection Tests
+          - name: awx-collection-sanity
+            command: /start_tests.sh test_collection_sanity
+            label: Run Ansible core Collection Sanity tests
```

Comment:
Not directly related to your PR, but I just noticed this label never gets used.

On (new) line 69, we have a typo: `matrix.texts.label` instead of `matrix.tests.label`.

Reply:
I have some follow-on changes to attempt, I might combine this comment with those.